libraries/AP_GPS/AP_GPS.h

```diff
@@ -392,7 +392,10 @@ class AP_GPS
     AP_Float _blend_tc;
 
 private:
-    // return gps update rate in milliseconds
+    // returns the desired gps update rate in milliseconds
+    // this does not provide any gurantee that the GPS is updating at the requested
+    // rate it is simply a helper for use in the backends for determining what rate
+    // they should be be configuring the GPS to run at
```

Comment:
/me whistles

I yanked from one buffer to the other, but then I fixed the line length in the one and just typed the changes into the other.... Anyways fixed.